wremnants/syst_tools.py

```diff
@@ -0,0 +1,21 @@
+import hist
+import numpy as np
+
+def scale_hist_to_variations(scale_hist):
+
+    s = hist.tag.Slicer()
+    # select nominal QCD scales, but keep the sliced axis at size 1 for broadcasting
+    nom_scale_hist = scale_hist[{"muRfact" : s[1.j:1.j+1], "muFfact" : s[1.j:1.j+1]}]
+
+    # select nominal QCD scales and project down to nominal axes
+    nom_hist = nom_scale_hist[{"ptVgen" : s[::hist.sum], "chargeVgen" : s[::hist.sum], "helicity" : s[::hist.sum], "muRfact" : s[1.j], "muFfact" : s[1.j] }]
```

Comment:
The slicer stuff is cool but if I read what you're doing right, wouldn't it be more clear to do nom_scale_hist[{"muRfact" : 1.j, "muFfact" : 1.j}].project("pt", "eta", "charge"). Then I guess you would need to expand the dimensions?